migrate/v0_16/testdata/genesis-v16.json

```diff
@@ -793,6 +793,117 @@
       "redelegations": [],
       "exported": false
     },
+    "kavadist": {
+      "params": {
+        "active": true,
+        "periods": [
+          {
+            "end": "2021-06-01T14:00:00Z",
+            "inflation": "1.000000000000000000",
+            "start": "2020-06-01T14:00:00Z"
+          },
+          {
+            "end": "2022-06-01T14:00:00Z",
+            "inflation": "1.000000002293273137",
+            "start": "2021-06-01T14:00:00Z"
+          },
+          {
+            "end": "2023-06-01T14:00:00Z",
+            "inflation": "1.000000001167363430",
+            "start": "2022-06-01T14:00:00Z"
+          },
+          {
+            "end": "2024-06-01T14:00:00Z",
+            "inflation": "1.000000000782997609",
+            "start": "2023-06-01T14:00:00Z"
+          }
+        ]
+      },
+      "previous_block_time": "2021-11-05T21:13:12.856088470Z"
```

Comment:
v15 is `2021-11-05T21:13:12.85608847Z`
There's an extra decimal point here.

Reply:
I think I found the reason. In the proto json spec: https://developers.google.com/protocol-buffers/docs/proto3#json it says the number of digits after the decimal point should always be 0, 3, 6, or 9. So I imagine gogoproto is adding an extra 0 to comply.

Reply:
oh nice, great find!